.github/workflows/build-binary.yml

```
@@ -0,0 +1,149 @@
+          --java-options '-Dfile.encoding="utf-8"'
+          ${JPACKAGE_OS_OPTS}
+      - uses: actions/upload-artifact@v4
+        with:
+          path: .\target\cryptomator-cli${{ matrix.binary-dir-suffix }}
+          if-no-files-found: error
+      - name: Zip binary for release
+        if: startsWith(github.ref, 'refs/tags/') && github.event.action == 'published'
+        shell: pwsh
+        run: Compress-Archive -Path .\target\cryptomator-cli${{ matrix.binary-dir-suffix }} -DestinationPath .\cryptomator-cli-${{ needs.prepare.outputs.semVerStr }}.zip
+      - name: Create detached GPG signature with key 615D449FE6E6A235
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1001:info:2:129: This \c will be a regular 'c' in this context

(shellcheck)